bundles/org.openhab.binding.velbus/src/main/java/org/openhab/binding/velbus/internal/handler/VelbusVMBMeteoHandler.java

```diff
@@ -107,6 +123,10 @@ protected int getClockAlarmAndProgramSelectionIndexInModuleStatus() {
     public void onPacketReceived(byte[] packet) {
         super.onPacketReceived(packet);
 
+        if (disposed) {
```

Comment:
Is there really no better way to implement this than to have each subclass track their disposed status? Could `onPacketReceived` be made to return a boolean and if the subclass sees it returning a `false` it could return early too?

Reply:
You're right, I will change this.

Reply:
Updated in last commit.